src/django_components/template.py

```diff
@@ -170,30 +170,45 @@ def _maybe_bind_template(context: Context, template: Template) -> Generator[None
 loading_components: List["ComponentRef"] = []
 
 
-def load_component_template(component_cls: Type["Component"], filepath: str) -> Template:
-    if component_cls._template is not None:
-        return component_cls._template
+def load_component_template(
```

Comment:
Previously this function set `Component._template`. Since that attribute is now managed in `component_media.py` , this function only returns the value without assigning it to the class